desktop-exporter/otlp_payload_processor.go

```diff
@@ -8,26 +8,54 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 )
 
+type ResourceData struct {
+	Attributes             map[string]interface{}
+	DroppedAttributesCount uint32
+}
+
+type ScopeData struct {
+	Name                   string
+	Version                string
+	Attributes             map[string]interface{}
+	DroppedAttributesCount uint32
+}
+
 type SpanData struct {
-	TraceID      string
+	TraceID    string
+	TraceState string
+
 	SpanId       string
 	ParentSpanID string
 	Name         string
+	Kind         string
 	StartTime    time.Time
 	EndTime      time.Time
-	Attributes   map[string]interface{}
-	Resource     *ResourceData
-	Scope        *ScopeData
+
+	Attributes map[string]interface{}
+	Events     []EventData
+	Links      []LinkData
+	Resource   *ResourceData
+	Scope      *ScopeData
+
+	DroppedAttributesCount uint32
+	DroppedEventsCount     uint32
+	DroppedLinksCount      uint32
+
+	StatusCode    string
+	StatusMessage string
```

Comment:
This is just to say that the way you broke up or ordered your attributes here makes intuitive sense to me and is pleasing and nice to read, but I could not tell you why

Reply:
nit: Maybe `SpanID` and `ParentSpanID` should be grouped with `TraceID` and `TraceState` though